src/category_theory/limits/shapes/zero.lean

```diff
@@ -194,14 +194,40 @@ by ext
 end
 
 /-- A zero object is in particular initial. -/
-def has_initial_of_has_zero_object : has_initial.{v} C :=
+def has_initial [has_zero_object.{v} C] : has_initial.{v} C :=
```

Comment:
It looks like the `[has_zero_object.{v} C]` is not needed here as it is already declared in line 142.

Reply:
Fixed, thanks.